service/src/intTest/resources/application.yml

```diff
@@ -40,13 +40,11 @@ gp2gp:
     ttl: ${GP2GP_MONGO_TTL:P7D}
     cosmosDbEnabled: ${GP2GP_COSMOS_DB_ENABLED:false}
   gpc:
-    url: ${GP2GP_GPC_GET_URL:https://orange.testlab.nhs.uk/B82617/STU3/1/gpconnect}
+    url: https://orange.testlab.nhs.uk/B82617/STU3/1/gpconnect
     structuredEndpoint: ${GP2GP_GPC_GET_STRUCTURED_ENDPOINT:/fhir/Patient/$gpc.getstructuredrecord}
     documentEndpoint: ${GP2GP_GPC_GET_DOCUMENT_ENDPOINT:/fhir/Binary/}
     patientEndpoint: ${GP2GP_GPC_GET_PATIENT_ENDPOINT:/fhir/Patient}
+    patientDocumentReferenceEndpoint: ${GP2GP_GPC_GET_DOCUMENT_ENDPOINT:/fhir/Patient}
```

Comment:
The default value is for a Patient resource not a DocumentReference resource

Reply:
Yes, but this url is used for open test. In the code we are adding "/document" prefix. Shall I move it here and remove "/document" concatenation in the code?